gulpfile.ts

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-import * as gulp from 'gulp';
+import gulp from 'gulp';
 import * as packageDependencyUpdater from './src/tools/UpdateRuntimeDependencies';

```

Comment:
did not change this one, seems the default export is not as expected

Reply:
change it into  
 `import {updateRuntimeDependencies} from './src/tools/UpdateRuntimeDependencies';`?

Reply:
found out as well